conda/recipes/libcudf/meta.yaml

```
@@ -105,6 +108,12 @@ outputs:
         - librmm ={{ minor_version }}
         - libkvikio ={{ minor_version }}
         - dlpack {{ dlpack_version }}
+        # TODO: start taking libarrow's run exports again wwhen they're correct for 16.0
+        # ref: https://github.com/conda-forge/arrow-cpp-feedstock/issues/1418
+        - libarrow>=16.0.0,<16.1.0a0
+        - libarrow-acero>=16.0.0,<16.1.0a0
+        - libarrow-dataset>=16.0.0,<16.1.0a0
+        - libparquet>=16.0.0,<16.1.0a0
```

Is the `libarrow` pin sufficient? The rest are pinned exactly to it (for [example]( https://github.com/conda-forge/arrow-cpp-feedstock/blame/d8df6c3c733fd88e0c4caebba652c0d0ba992f7e/recipe/meta.yaml#L291-L292 ))

```suggestion
        # TODO: start taking libarrow's run exports again wwhen they're correct for 16.0
        # ref: https://github.com/conda-forge/arrow-cpp-feedstock/issues/1418
        - libarrow>=16.0.0,<16.1.0a0
```

---

You're probably right that it would be enough. I went for all of them just mimicking what I already saw in `dependencies.yaml` here for building the test environment.

Alright if we still merge this, and then i push a follow-up PR trying to pare this back down to just `libarrow`? That way we wouldn't have to wait for another CI cycle to get the fix onto `branch-24.06`.

---

I don't think it's worth changing at this point. Once the upstream arrow packages are updated we'll just be dropping this altogether anyway.

---

Ok works for me, I won't touch this unless @jakirkham disagrees and thinks we should pursue it.